apps/portal/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
-    "strict": true,
```

Comment:
Why did we have to change tsconfig?
The base setup was made using the angular standards.

Reply:
This was done to implement the Apollo client, I have taken reference from the Jet Luxury project.

I was facing various difficulties while implementing the Apollo client.

Also in angular standards, nothing was mentioned regarding tsconfig.json, thus modified it earlier.

https://github.com/OsmosysSoftware/dev-standards/blob/main/coding-standards/angular.md